### PR TITLE
Bitcoin Core 25.1

### PR DESCRIPTION
**Pull request by fanquake**
Release annoucement for `25.1`.

### CODE REVIEW

File: _posts/en/posts/2023-10-15-release-25.1.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  Bitcoin Core 25.1 is now available.
+---
+Bitcoin Core version 25.1 is now available for [download][download
+page].  See the [release notes][] for more information about the many
+bug fixes in this release.
+
+If you have any questions, please stop by the #bitcoin IRC chatroom
+([IRC][irc], [web][web irc]) and we’ll do our best to help you.
+
+[release notes]: /en/releases/24.1/
```
Comment (katesalazar) [inline _posts/en/posts/2023-10-15-release-25.1.md:25]: 24.1

**Review by katesalazar — CHANGES_REQUESTED** [review]

**Comment by fanquake** [comment]
Binaries are now available: https://bitcoincore.org/bin/bitcoin-core-25.1/.
Updated with the magnet URL.

**Review by glozow — COMMENTED** [review]
ACK ae5bd17

built preview, looks fine, text looks correct, links seem to work

File: _releases/25.1.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+---
+title: Bitcoin Core 25.1
+id: en-release-25.1
+name: release-25.1
+permalink: /en/releases/25.1/
+excerpt: Bitcoin Core version 25.1 is now available
+date: 2023-10-16
```
[the remark this answers is not in the document]
Comment (fanquake) [reply _releases/25.1.md]: Fixed.

**Review by glozow — COMMENTED** [review]
reACK b6650ed

**Pull request merged**